lib/index.ts

```diff
@@ -40,7 +41,7 @@ function registerGlobal(name: string) {
       exports, name, {get: () => exports.protractor ? exports.protractor[name] : undefined});
 }
 
-export declare let browser: ProtractorBrowser;
+export declare let browser: ProtractorBrowser&WebDriver;
```

Comment:
I missed the `&` the first time I read it. Maybe `ProtractorBrowser&WebDriver` -> `ProtractorBrowser & WebDriver`?

Reply:
I think the same interface update will be needed on [`Ptor.browser`](https://github.com/angular/protractor/blob/4bc80d1a459542d883ea9200e4e1f48d265d0fda/lib/ptor.ts#L15) field.

That's for people referencing `browser` from `protractor` object:

```typescript
import { protractor } from 'protractor';

protractor.browser.someMethod()
```

Reply:
Initially I using syntax as you described, but got tslint error which is included into Protractor gulp pre-publish.
So this is due to tslint. I think we need to remove it and replace within the modern eslint. Regarding "Ptor" I agree, will add this.

Reply:
Here is very unusefull clang-format script under gulp "lint" task. I think need to revise it and totally replace with eslint.